service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseNoteServiceTest.java

```diff
@@ -24,23 +27,32 @@
 import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.unwrapElements;
 import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
 
-@ExtendWith(MockitoExtension.class)
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = { CaseNoteService.class, FixedTimeConfiguration.class})
 class CaseNoteServiceTest {
 
-    @Mock
+    @Autowired
+    private Time time;
+
+    @MockBean
     private IdamClient idamClient;
 
-    @InjectMocks
     private CaseNoteService service;
 
     private static final String userAuthToken = "Bearer";
     private static final UserInfo userDetails = UserInfo.builder().name("John Smith").build();
 
+    @BeforeEach
+    void setUp() {
+        service = new CaseNoteService(idamClient, time);
+    }
+
     @Nested
     class BuildCaseNote {
 
         @BeforeEach
-        void setup() {
+        void setUp() {
+            service = new CaseNoteService(idamClient, time);
```

Comment:
This isn't necessary as the outer for each is called before this one

Reply:
yup will remove